development-docs/systemtests/labels/plaintext.md

```diff
@@ -0,0 +1,24 @@
+# Plaintext
+
+The `plaintext` label focuses on testing communication mechanisms over unencrypted PLAINTEXT channels. 
```

Comment:
> Guess it will cover in the end just a several tests as well or?

I mean plaintext will have a lot of tests I think. In other tests, where we use unencrypted messages :)

Reply:
For now updated to have just one in this package. In other packages we will have more (e.g., upgrade, downgrade, auth, authz, certificate exchange)